codexctl/device.py

```
@@ -0,0 +1,643 @@
+            print("Done! Now rebooting the device and disabling update service")
+
+            self.client.exec_command("sleep 1 && reboot")  # Should be enough
+            self.client.close()
+
+            time.sleep(
+                2
+            )  # Somehow the code runs faster than the time it takes for the device to reboot
+
+            print("Trying to connect to device")
+
+            while not self.check_is_address_reachable(address):
```

Ruff (0.8.0)

Undefined name `address`

(F821)